### ISSUE TITLE
Song doesn't play.

### ISSUE THREAD

**Issue by KABYb34** [fast4x/RiMusic#4274]
### Version of RiMusic

0.6.58.1

### Android version

14

### I've done these:

- [X] Uninstall old build before installing a new one
- [X] Checked opening issues about my similar problem

### Tell us how you trigger this bug

1. Open RiMusic
2. Click on song
3. Doesn't work

### Submit a video

If you are on Android, please download a screen recorder and record your screen while you
duplicating the problem as describe above.

> You may need to use Github App to be able to upload video - size limit is 10 MB.

If you are on Desktop, drag video into this box and it will be uploaded automatically

### What was your expectation?

For a song to play

### What more can you tell us? (Optional)

Player sometimes works and sometimes doesn't even with trouble shooting.  Btw love yalls work :)

**Comment by fast4x** [comment]
try to login to ytm from miscellaneous settings

**Comment by KABYb34** [comment]
Yes i tried that but that doesn't work, i log in and then nothing, tried multiple times restarted every time but nothing.

**Comment by fast4x** [comment]
wait next version

**Comment by KABYb34** [comment]
Alr :)

**Comment by fast4x** [comment]
Try custom version please download https://github.com/fast4x/RiMusic/tree/master/customVersion

**Comment by paulpremont** [comment]
It works with this custom version, thanks ! (same problem before : Song cannot be played with the official 0.6.58.1 + Android 13).

**Comment by iskyd** [comment]
Same here with Android 14, unable to reproduce songs with latest version from fdroid but it works with custom version.

**Comment by dreamAviator** [comment]
Will it be mandatory to log in to ytm from now on?

**Comment by fast4x** [comment]
Wait for next release, there's an improvement.

**Comment by fast4x** [comment]
> Same here with Android 14, unable to reproduce songs with latest version from fdroid but it works with custom version.

Yes, FDroid version is published by FDroid after one or two weeks.
